src/data/admin/Mentors.tsx

```diff
@@ -61,6 +107,7 @@ export const DROPDOWN = ({ object }) => {
               toggle={object.availability.includes(text)}
               text={text}
               key={index}
+              color
```

Comment:
this should not be needed

Reply:
`Property 'color' is missing in type '{ toggle: any; text: string; key: number; }' but required in type '{ toggle: any; onClick?: (() => void) | undefined; text?: string | undefined; color: any; `